examples/elf/tests/mutex/mutex.c

```diff
@@ -103,7 +103,7 @@ int main(int argc, char **argv)
 
   printf("Starting thread 1\n");
   bendoftest = false;
-  if ((pthread_create(&thread1, NULL, (void*)thread_func, (void*)1)) != 0)
+  if ((pthread_create(&thread1, NULL, (void *)thread_func, (void *)1)) != 0)
```

Comment:
Can we fix ling line warning at line 45?

Reply:
Fix